### ISSUE TITLE
Why the flow values are not normalized to [-1, 1] ?

### ISSUE THREAD

**Issue by sangenan** [MCG-NKU/E2FGVI#18]
Thank you for your answer

**Comment by NK-CS-ZZL** [comment]
Thanks for your Question! Could you give me a more specific description? I can't fully understand your question through a single sentence. I guess you have questions about the format of inputs or outputs of SpyNet which is used to estimate optical flow. The inputs of SpyNet should be normalized into [0, 1] aligned with pretrained model. The outputs of SpyNet are the estimated optical flow which represents 2D displacement in pixel level and is unreasonable to constrain into [0, 1]. If optical flow is not a familiar concept to you, you can refer to [wikipedia](https://en.wikipedia.org/wiki/Optical_flow).

**Comment by sangenan** [comment]
I'm sorry that my question is not very clear. There is a function called `flow_warp (x,
              flow,
              interpolation='bilinear',
              padding_mode='zeros',
              align_corners=True) `in the file named flow_comp.py.  The input flow values why not normalized to [-1, 1]. 
Due to my carelessness, I didn't notice normalized operations in the function.
Thank you for your reply.

**Comment by NK-CS-ZZL** [comment]
I got what you mean. To match the interface of `grid_sample` in pytorch, the flow ought to be in range [-1, 1]. 
If you have any questions, feel free to contact us. XD
